.github/CODEOWNERS

```diff
@@ -9,45 +9,60 @@
 /sdk/                                           @AlexGhiondea
 
 # Core
+# PRLabel: %Azure.Core
 /sdk/core/                                      @pakrym @KrzysztofCwalina
 
 # Extensions
 /sdk/extensions/                                @parkym
 
 # Service teams
+# PRLabel: %AzConfig
 /sdk/appconfiguration/                          @annelo-msft @AlexanderSher
 
+# PRLabel: %Batch
 /sdk/batch/                                     @bgklein @xingwu1 
 
 /sdk/cognitiveservices/language.textanalytics/  @assafi 
 /sdk/cognitiveservices/vision.computervision/   @toothache
 /sdk/cognitiveservices/vision.face/             @TFR258 
 
+# PRLabel: %DigitalTwins
 /sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @prmathur-microsoft @bikamani @barustum
 
+# PRLabel: %Event Grid
 /sdk/eventgrid/                                 @Kishp01 @ahamad-MS
 
+# PRLabel: %Event Hubs
```

Comment:
Not your addition, but I don't think we need line 37;  I don't remember why we started down that path, but line 36 covers that.

Reply:
Fixed.